components/brave_wallet_ui/common/slices/endpoints/nfts.endpoints.ts

```diff
@@ -242,6 +255,44 @@ export const nftsEndpoints = ({
         }
       },
       providesTags: ['AutoPinEnabled']
+    }),
+    getIPFSUrlFromGatewayLikeUrl: query<string | null, string>({
+      queryFn: async (urlArg, store, extraOptions, baseQuery) => {
+        try {
+          const { cache } = baseQuery(undefined)
+          const ipfsUrl = await cache.getExtractedIPFSUrlFromGatewayLikeUrl(
+            urlArg
+          )
+          return {
+            data: ipfsUrl || null
+          }
+        } catch (error) {
+          const message =
+            'Failed to ge IPFS URL from gateway-like URL: ' + urlArg.trim()
+          console.log(message, error)
```

Comment:
The way to avoid these possible injections would be to do:
```
console.log('Failed to get IPFS URL from gateway-like URL: %s', urlArg.trim(), error)
```

Reply:
fixed